puppet/modules/scaleway/manifests/cron.pp

```diff
@@ -0,0 +1,24 @@
+# Deploys a cronjob which calls the Scaleway API to
+# snapshot a server. Does not need to be on the Scaleway host.
+#
+define scaleway::cron (
+  String $api_key                   = 'NA',
+  String $org_id                    = 'NA',
+  Enum['present', 'absent'] $ensure = present,
+) {
+  file { "/etc/scaleway/${name}.yaml":
+    ensure  => $ensure,
+    owner   => 'root',
+    group   => 'root',
+    mode    => '0700', #root only, contains API keys
+    content => "api_key: <%= @api_key -%>\norg_id: <%= @org_id  -%>\n",
+  }
+
+  cron { "scaleway-snapshot-${name}":
```

Comment:
This probably also needs `ensure => $ensure` so we can purge old ones.

Reply:
good catch, ty